includes/Settings.php

```diff
@@ -60,6 +60,34 @@ class Settings {
 	 */
 	const SETTING_NAME_TRACKING_ID = 'web_stories_ga_tracking_id';
 
+	/**
+	 * Type of adloader.
+	 *
+	 * @var string
+	 */
+	const SETTING_NAME_ADLOADER_TYPE = 'web_stories_adloader_type';
```

Comment:
Let's use `ad network` instead of `adloader` everywhere throughout the codebase. So:

```suggestion
	/**
	 * Chosen ad network.
	 *
	 * @var string
	 */
	const SETTING_NAME_AD_NETWORK = 'web_stories_ad_network';
```

Reply:
Fixed in 
https://github.com/google/web-stories-wp/pull/5611/commits/5cc53787b274aeb0499950a67c1a297ec177e96f
https://github.com/google/web-stories-wp/pull/5611/commits/20709b00accc04d970dd27187022cb82edb3d2d5